pkgs/applications/misc/writefreely/default.nix

```diff
@@ -13,12 +13,37 @@ buildGoModule rec {
 
   vendorSha256 = "sha256-CBPvtc3K9hr1oEmC+yUe3kPSWx20k6eMRqoxsf3NfCE=";
 
+  writefreely-assets = stdenv.mkDerivation {
```

Comment:
It would be great to put it in `passthru` so we can do `writefreely.writefreely-assets` and you can still refer to it in this drv.